src/client/components/FavouriteList/FavouriteList.component.js

```diff
@@ -5,12 +5,14 @@ import ProductImage from './ProductImage.js';
 import ProductQuantity from './ProductQuantity';
 import ProductCart from './ProductCart';
 
-export default function FavouriteList({ productList }) {
+export default function FavouriteList( props ) {
+  const {favorites,setFavorites}=props;
+  console.log(favorites)
   return (
-    <>
-      <h2 className="heading_div">Favourites</h2>
-      <ul className="container">
-        {productList.map((product) => (
+    <div>
+      {/* <h2 className="heading_div">Favourites</h2> */}
+      <ul className="favorite-container">
+        {favorites.map((product) => (
```

Comment:
This are changes from another pr/branch ...  I think something happened between jumping from branch to branch

Reply:
@suvarna-ratna Please explain that.